src/main/windows/mainWindow.ts

```diff
@@ -246,6 +249,13 @@ export class MainWindow extends EventEmitter {
         return os.platform() === 'darwin' || (os.platform() === 'win32' && Utils.isVersionGreaterThanOrEqualTo(os.release(), '6.2'));
     };
 
+    private getTitleBarOverlay = () => {
+        return {
+            color: Config.darkMode ? '#2e2e2e' : '#efefef',
+            height: TAB_BAR_HEIGHT,
```

Comment:
Does just by increasing height we have the button? i dont see where is the part of the buttons?

Reply:
The `titleBarOverlay` field takes in an object of type `Object | boolean`. So just by having the object there it evaluates to `true` and under the hood Electron uses these fields to construct the buttons.

Reply:
Ah i see